cartridges/plugin_gtm/cartridge/scripts/gtm/gtmHelpers.js

```diff
@@ -47,7 +48,7 @@ function getCustomerData(res) {
  */
 function getHomeData() {
     var obj = {
-        'event': 'home'
```

Comment:
let's not remove the single quotes on properties.

Reply:
I undid all the code formatting/linting